server/internal/usecase/interactor/project.go

```diff
@@ -47,6 +47,35 @@ func (i *Project) Create(ctx context.Context, param interfaces.CreateProjectPara
 	if !op.IsUserOrIntegration() {
 		return nil, interfaces.ErrInvalidOperator
 	}
+
+	visibility := project.VisibilityPrivate
```

Comment:
```suggestion
	visibility := project.VisibilityPublic
```
The default visibility is public.